plugin.xml

```diff
@@ -74,6 +74,9 @@
         <pod name="Fabric"/>
         <pod name="Crashlytics" />
         <pod name="FirebaseCore" />
+        <pod name="Firebase/Analytics" />
+        <pod name="Firebase/Auth" />
+        <pod name="Firebase/Firestore" />
```

Comment:
Are these required for Crashlytics?
I'd say that for example Firebase/Firestore is a database not related to this context

Reply:
> Are these required for Crashlytics?
> I'd say that for example Firebase/Firestore is a database not related to this context

They are from the Firebase documentation ([https://firebase.google.com/docs/ios/setup?authuser=1](https://firebase.google.com/docs/ios/setup?authuser=1)) and they shouldn't be required but I have fatal error in ionic build command without these lines, so...

Reply:
I see

Reply:
Could you try with just `Firebase/Analytics` as described in https://firebase.google.com/docs/ios/setup?authuser=1#add-sdks?

There is another similar PR #30 which adds only `Firebase/Analytics` dependecy

Reply:
> Could you try with just `Firebase/Analytics` as described in https://firebase.google.com/docs/ios/setup?authuser=1#add-sdks?
> 
> There is another similar PR #30 which adds only `Firebase/Analytics` dependecy

It works with only `Firebase/Analytics`. I commited this change to update my PR